src/components/FlatList/styled.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components/native';
+import { COLOR_GRAY_TRANSPARENT_1 } from '../../styles/colors';
+
+export const Divider = styled.View`
+    height: 1px;
+    margin-left: 20px;
+    width: 100%;
+    background-color: ${COLOR_GRAY_TRANSPARENT_1};
```

Comment:
we are doing this in the app, but really we should get it from props.theme ... like we do in react rainbow in order to get this working with theming